api-testing/multicloud-api/README.md

```diff
@@ -0,0 +1,9 @@
+# Multi cloud API Automation
+
+**Following tool you can use for this test automation**
+
+ Intellij idea community edition [link](https://www.jetbrains.com/idea/download/#section=windows)
+
+**How to import code in intellij idea community edition?**
```

Comment:
The readme looks incomplete.
Please give the introduction to framework
why this, how to install and use
how to add tests to this etc...
If you have another guide for this, you can link it here.

Reply:
I have changed as per your comment . Please review

Reply:
@Click2cloud-Wadhai Still not having the relevant info. Added another comment on the new update